src/jmh/java/com/datadoghq/sketch/ddsketch/Distributions.java

```diff
@@ -5,22 +5,26 @@
 
 package com.datadoghq.sketch.ddsketch;
 
-import java.util.concurrent.ThreadLocalRandom;
+import java.util.Random;
 
 public enum Distributions {
   NORMAL {
     @Override
     protected Distribution create(double... parameters) {
-      return () -> parameters[1] * ThreadLocalRandom.current().nextGaussian() + parameters[0];
+      final Random random = new Random(seed);
```

Comment:
Good catch. `SplittableRandom`  generates better quality output and takes a seed, it could be used here.

Reply:
I didn't know about `SplittableRandom `, thanks for the tip. However, it doesn't have `nextGaussian()`. I could copy the implementation, but for simplicity, I think I'll keep `Random` (unless the output quality raises issues).